### ISSUE TITLE
Readme.md is outdated

### ISSUE THREAD

**Issue by DimatiBart** [redux-loop/redux-loop#127]
Nuff said. Trying to use redux-loop, and can't find any examples of usage. Npm readme is missing, and even readme in the master branch of this repo is 50% useless. And after couple of hours I found out that the only good doc is in 3.0 branch. Could you guys please update the readme or even merge the master with 3.0?

**Comment by timkendall** [comment]
Yeah it's too bad, this is a pretty sweet project. Just seems like it's fallen behind a little. Maybe the core contributors have fully moved on to Elm :)

In any case, @lukewestby would love to help out if there were some like "required-for-3.0-official-release" issues we could knock out.

**Comment by ashishsc** [comment]
I would be happy to help get this project going again, @lukewestby is there anything that we can do to help move things along?

**Comment by bdwain** [comment]
Hi everyone.

Sorry it's been so long since you've heard back. I'm going to be taking over maintenance of redux-loop. I'd like to get an update out in the next few days.

I know there have been a few proposed API changes in the last year. I've been using a fork of one those proposed version 3's for a while in projects that I've worked on (https://github.com/bdwain/redux-loop). I'd like to make that the starting point for any future changes. Luke and I discussed these changes last year and I think it'll be an improvement over version 2 without being too drastic of a change for people who are currently using it. It doesn't have typescript typings yet, but I would like to add that in the next few days as well.

The main idea behind the changes in my branch can be seen here [in this issue](https://github.com/redux-loop/redux-loop/issues/66) . It's passing action creators to effects/cmds instead of putting them in your side effect functions. Though since that discussion, i've consolidated some of the Cmd types.

I'll keep this issue open for now and close it when I've got everything updated. Sorry again for the delay!
